src/org/ods/orchestration/usecase/JiraUseCase.groovy

```diff
@@ -110,14 +116,18 @@ class JiraUseCase {
         if (!this.jira) return
 
         testFailures.each { failure ->
-            def bug = this.jira.createIssueTypeBug(this.project.jiraProjectKey, failure.type, failure.text)
+            def bug = this.jira.createIssueTypeBug(
+                this.project.jiraProjectKey, failure.type, failure.text, this.project.versionFromReleaseStatusIssue)
```

Comment:
`this.project.versionFromReleaseStatusIssue` runs a several queries against Jira. Why not use the `Project.getVersionName()`?

Reply:
We should not use `this.project.versionFromReleaseStatusIssue` or other methods which load data from Jira more then once. Currently, is it in use when loading versioned data in Project.groovy.

Reply:
As this is Groovy, and we can not hide things with 'private', another solution that I saw is to create a sub-package with the name "private", for the things that we don't want other code to use.

Reply:
@s2oBCN @jorge-romero I'd consider it good practice to load any Jira data in `Project.load` and not load any data multiple times. Unfortunately, versions are loaded in Project.groovy and JiraUseCase.groovy. This needs to be consolidated in the next version. I'll add `FIXME` comments to remind us.